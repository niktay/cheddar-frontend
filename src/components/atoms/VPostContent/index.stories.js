import { storiesOf } from "@storybook/vue";
import VPostContent from ".";

storiesOf("Atom - VPostContent", module).add("default", () => ({
  components: { VPostContent },

  template: `
  <VPostContent>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptate mollitia quaerat voluptatem optio obcaecati 
    quos ex saepe. Minus, optio quis! Labore fugit deleniti ex id commodi ab nobis eaque adipisci?
  </VPostContent>`
}));
