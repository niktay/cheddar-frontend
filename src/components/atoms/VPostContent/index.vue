<template>
  <p class="a-VPostContent">
      <slot></slot>
  </p>
</template>

<script>
export default {};
</script>

<style class="scss">
.a-VPostContent {
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  font-size: 1.125rem;
  line-height: 2.25rem;
  color: #454545;
}
</style>
