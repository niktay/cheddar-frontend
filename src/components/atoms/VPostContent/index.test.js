import { mount } from "@vue/test-utils";
import VPostContent from ".";

describe("VPostContent component", () => {
  it("should render correctly", () => {
    const comp = mount(VPostContent, {
      slots: {
        default: "The quick brown fox jumps over the lazy dog"
      }
    });

    expect(comp.html()).toMatchSnapshot();
  });
});
