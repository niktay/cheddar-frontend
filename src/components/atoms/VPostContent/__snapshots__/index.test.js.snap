// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VPostContent component should render correctly 1`] = `<p class="a-VPostContent">The quick brown fox jumps over the lazy dog</p>`;
